ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -118,6 +118,9 @@ elif [ "$PX4_SIMULATOR" = "gz" ] || [ "$(param show -q SIM_GZ_EN)" = "1" ]; then
 			echo "ERROR [init] gz_bridge failed to start and spawn model"
 			exit 1
 		fi
+		sleep 1 # Needed to wait for the model to spawn
+		gz service -s /gui/follow --reqtype gz.msgs.StringMsg --reptype gz.msgs.Boolean --timeout 2000 --req "data: \"${PX4_SIM_MODEL#*gz_}_0\""
+		gz service -s /gui/follow/offset --reqtype gz.msgs.Vector3d --reptype gz.msgs.Boolean --timeout 2000 --req "x: 2, y: 1, z: 1"
```

Comment:
```suggestion
		gz service -s /gui/follow/offset --reqtype gz.msgs.Vector3d --reptype gz.msgs.Boolean --timeout 2000 --req "x: -6, y: 0, z: 6"
```
Can we leave the requested pose at the initial camera pose defined in the default sdf so there is no rotation upon start up?